packages/integration-tests/suites/replay/customEvents/template.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+  </head>
+  <body>
+    <!-- <button id=""onclick="console.log('Test log')">Click me</button> -->
```

Comment:
Should this be removed?

Reply:
Yup, removed [17c28a0](https://github.com/getsentry/sentry-javascript/pull/7052/commits/17c28a004f576973ecda958139e9327052d18df8)